src/cli.rs

```diff
@@ -2,27 +2,21 @@ use std::{ffi::OsString, fmt};
 
 use clap::{ArgEnum, Parser, Subcommand};
 
-#[derive(ArgEnum, Copy, Clone, Debug, PartialEq)]
+#[derive(ArgEnum, Copy, Clone, Default, Debug, PartialEq)]
 pub enum Format {
+    #[default]
```

Comment:
Did these changes need to be made? As this increases the MSRV of the project.

Reply:
Functionally no, but they will be necessary to appease Clippy, i.e. the `lint` CI job, eventually. As I saw them pending in other PR as well and did not find any explicit MSRV information, I thought to include them.

I suspect we could drop them for now as stable Clippy is used in the CI though.

Reply:
@adamreichold that is what I would do.  keep the changes here focused just on the need of bumping cargo.  And deal with things that change MSRV to a separate PR.

(as I would really like to get an updated cargo-outdated in place so I can use sparse registries :) )